aspnetcore/includes/spa-templates-new-vs-old.md

```diff
@@ -0,0 +1,25 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/12/2023
+ms.topic: include
+---
+## Newer vs. older SPA templates
+
+The SPA templates that use the special project type for JavaScript and TypeScript are available in Visual Studio 2022 version 17.7 or later with the **ASP.NET and web development** workload installed. They're available only in Visual Studio, not by using the `dotnet new` command of the .NET CLI.
+
+There are older SPA templates that don't use the special project type for JavaScript and TypeScript. The newer Visual Studio templates offer the following benefits compared to the older templates:
```

Comment:
Can we reword this part to not be a comparison? In the long run the intention is that the new templates fully replace the old ones. Can we make the language here timeless?

```suggestion
The Visual Studio templates for building ASP.NET Core apps with a JavaScript or TypeScript frontend offer the following benefits:
```